dotcom-rendering/src/web/components/MatchTabs.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/react';
+import { border } from '@guardian/src-foundations/palette';
+import { space, palette } from '@guardian/src-foundations';
+import { headline } from '@guardian/src-foundations/typography';
+
+type Props = {
+	minByMinUrl?: string;
+	matchUrl?: string;
```

Comment:
```suggestion
	reportUrl?: string;
```

Reply:
I thought we needed the match URL?

Reply:
Isn't this a different url? I thought `matchUrl` is the api endpoint and then the `MatchTabs` component wants the page address for the match report?

But maybe this is just a naming glitch?